content/doc/developer/plugin-governance/managing-permissions.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: developersection
+title: Managing plugin permissions
+references:
+- url: https://github.com/jenkins-infra/repository-permissions-updater
+  title: Repository Permissions Updater
+- url: ../adopt-a-plugin
+  title: Adopting a plugin
+---
+
+This page documents managing permissions for plugin maintainers.
+There are 2 main permissions available to plugin contributors:
+
+* **GitHub permissions**. 
+  It includes merging pull requests, push to the repositories, managing GitHub applications, etc.
+  The effective list of permissions depends on the link:https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization[permission levels], see below.
+* **Release Permissions** which are needed to deploy releases to the link:https://repo.jenkins-ci.org/webapp/#/home[Jenkins artifact repository].
+  Snapshots can be deployed by contributors without this permission.
+
+NOTE: If you want to adopt a plugin, please see link:../adopt-a-plugin[Adopt a Plugin] instead.
+
+== GitHub Permissions
+
+GitHub permissions in Jenkins are managed by GitHub teams.
+Almost every plugin has a `<pluginId> Developers` team created specifically for this plugin.
+This team may have a different link:https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization[permission level] depending on time of creation.
+For all new repositories we tend to grant `Admin` permissions at the moment, and other plugin teams can get permissions elevated upon a request.
+
+How to request GitHub permissions?
+You will need to send a permission transfer request and to specify the permission recipient's GitHub account and Jenkins LDAP(aka Jenkins Jira) account there.
+
+* Request permissions change in Jenkins Jira
+** Project: `INFRA`, Component: `github`
+* Request permissions in the link:https://groups.google.com/forum/#!forum/jenkinsci-dev[:Jenkins developer mailing list].
+
+Permission change requests should be ideally done by one of the active plugin maintainers.
+Otherwise, an explicit approval will be required before the permission transfer happens.
+If there are multiple active maintainers in a plugin, Jenkins GitHub administrators will usually wait for a while to collect approvals from a number of them.
```

Comment:
I think this section should explicitly distinguish between:

* An existing maintainer wants to add co-maintainers, committers, or transfer ownership to another user (perhaps because the other user asked them for it directly, but doesn't matter)
* Someone wants to take over an abandoned plugin / unresponsive maintainer

The former is a simple admin task, so it's good enough to have a PR to RPU, or a Jira issue filed by one of the RPU uploader accounts (or both). There is never a reason to go through the dev list for this.

The latter needs dev list involvement per adopt-this-plugin, etc. as it needs more visibility and awareness, as there is a greater potential for it to go wrong.

Reply:
I'd be fine with this change as is, or also fine with the additional information proposed by @daniel-beck .  I think this is a positive step whether or not it includes the recommended addition.